.github/auto-assignees.yml

```diff
@@ -0,0 +1,28 @@
+---
+# This assigns a PR to its author
+addAssignees: author
+
+reviewers:
+  # The default reviewers
+  defaults:
+    - maintainers
+
+  groups:
+    maintainers:
+      - carlisia
+      - jenting
+
+    other-maintainers:
+      - ashish-amarnath
+      - zubron
+      - dsu-igeek
+      - sseago
+
+options:
+  ignore_draft: true
+  ignored_keywords:
+    - WIP
+    - wip
+    - DO NOT MERGE
+  enable_group_assignment: true
+  number_of_reviewers: 3
```

Comment:
Do we still need 3 reviewers here? We're using 2 for the main velero project and the new auto-request-review action seems to handle that well (it looks like there was an issue with the previous action being used).

Reply:
Because this repo has the potential to be de-prioritized, I thought it would be best to require the reviews of 3 people.  I should've documented this in the PR. 

The 2nd reviewer who gets to it first can merge the PR. This is not requiring 3, only requesting 3. We still only require 2.

WDYT?

Reply:
I'd prefer using 2.
We could use Slack _or_ GitHub notification to ask other maintainers to review.

Reply:
Ok I'll change it to 2.